core/controllers/reader.py

```diff
@@ -468,6 +513,12 @@ class LearnerIncompleteActivityHandler(base.BaseHandler):
     """
     @acl_decorators.can_access_learner_dashboard
     def delete(self, activity_type, activity_id):
+        """Removes exploration or collection from incomplete list.
+
+        Args:
+            activity_type: str. The activity type.
```

Comment:
Can even add possible values it can take, 
`activity_type: str. The activity type. Currently, it can take values "exploration" or "collection".`